PartsAppTests/PartsAppTests.csproj

```diff
@@ -0,0 +1,29 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
why 6 and not 8?

Reply:
when tried to use 8.0, I got the error: "Unable to create instance of class TestProject1.UnitTest1+ExcelHelperTests. Error: System.BadImageFormatException: Could not load file or assembly '**...bin\Debug\net8.0\PartsApp.exe**'."

Reply:
So? resolve the issue.

Reply:
added test project of .NET Framework 4.8 (like other projects in the solution)